### PR TITLE
CNV-28176: Add CPU and memory hot-plug

### PR DESCRIPTION
**Pull request by pcbailey**
## 📝 Description

This PR adds UI support for CPU and memory hot-plug.

Jira (CPU): https://issues.redhat.com/browse/CNV-28176
Jira (Memory): https://issues.redhat.com/browse/CNV-33079

## 🎥 Screenshots

cpu-memory-hot-plug-AFTER-modal--2024-04-29_10-37

cpu-memory-hot-plug-AFTER-modal-title-popover--2024-04-29_10-38

cpu-memory-hot-plug-AFTER-modal-cpu-breakdown-popover--2024-04-29_10-40

### CODE REVIEW

File: src/utils/components/CPUMemoryModal/components/CPUInput/CPUHelperText.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+  return (
+    <div className="cpu-helper-text">
+      {t('Total CPU is {{totalCPU}}', { totalCPU: totalCPU })}
+      <Popover
+        bodyContent={
+          <Stack className="cpu-helper-text__body-content">
+            <StackItem>{t('{{sockets}} sockets', { sockets: sockets })}</StackItem>
+            <StackItem>{t('{{threads}} threads', { threads: threads })}</StackItem>
+            <StackItem>{t('{{cores}} cores', { cores: cores })}</StackItem>
```
Comment (metalice) [inline src/utils/components/CPUMemoryModal/components/CPUInput/CPUHelperText.tsx:38]: can be simply {socket } , { cores } , {threads}

File: src/utils/components/CPUMemoryModal/components/CPUInput/CPUMemoryModalHeader.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+const CPUMemoryModalHeader: FC = ({}) => {
+  const { t } = useKubevirtTranslation();
+
+  return (
+    <>
+      <Title
+        className="cpu-memory-modal-header"
+        headingLevel="h4"
+        id="cpu-memory-modal-title"
+        size={TitleSizes.md}
```
Comment (metalice) [inline src/utils/components/CPUMemoryModal/components/CPUInput/CPUMemoryModalHeader.tsx:18]: whats the difference between size and headinglevel?

Comment (pcbailey) [reply src/utils/components/CPUMemoryModal/components/CPUInput/CPUMemoryModalHeader.tsx:18]: I just looked. They map to each other. h4 maps to md, so I'll remove the size.

File: src/utils/components/CPUMemoryModal/components/CPUInput/CPUHelperText.tsx (hunk 1 above)
Comment (metalice) [reply src/utils/components/CPUMemoryModal/components/CPUInput/CPUHelperText.tsx:38]: should act the same

**Pull request merged**